deblur/workflow.py

```diff
@@ -427,6 +427,9 @@ def remove_artifacts_seqs(seqs_fp,
         else:
             sim_thresh = 0.65 * 100
 
+    # the minimal average bitscore per nucleotide
+    bitscore_thresh = 0.65
```

Comment:
I think it's better not to have it as a CLI parameter? We have lot's of parameters already, and this is just a convenience function for running SortMeRNA - so if a user wants finer control as to what's considered 16S/not 16S her can run SortMeRNA on all.biom with whatever database/parameters he wants and get the same thing...
But can also add the CLI parameter - not strong opinion - what do others think?